services/sidecar/rollup/services/disseminator/batch_disseminator.go

```diff
@@ -64,10 +64,11 @@ func (d *BatchDisseminator) start(ctx context.Context) error {
 		select {
 		case <-ticker.C:
 			if err := d.step(ctx); err != nil {
-				if errors.As(err, &unexpectedSystemStateError{}) {
-					return fmt.Errorf("aborting: %w", err)
-				}
 				log.Errorf("Failed to step: %w", err)
+				if errors.As(err, &recoverableSystemStateError{}) {
+					d.rollback(ctx)
```

Comment:
do some log here to see if this if-branch is triggerred

Reply:
added